server.js

```diff
@@ -151,7 +178,9 @@ scServer.on('connection', function (socket) {
     });
 
     respond();
-    console.log(`Server ${data.instanceId} at address ${socket.instanceIp} on port ${socket.instancePort} joined the cluster`);
+    if (LOG_LEVEL >= 3) {
+      console.log(`Server ${data.instanceId} at address ${socket.instanceIp} on port ${socket.instancePort} joined the cluster`);
```

Comment:
You could group all of those into a single `log` function to avoid repetition.

So you could define a more specific logging function like this (for example):
```
var log = function (message) {
  if (LOG_LEVEL >= 3) {
    console.log(message);
  }
};
```

Then from now on you don't need to keep checking the log level and you don't call console.log directly, you can just do: `log('Server ${data.instanceId} at address ${socket.instanceIp} on port ${socket.instancePort} joined the cluster')` and the log level will automatically be taken into account.

You can also follow a similar pattern for warn and error.